### ISSUE TITLE
I cannot move a object

### ISSUE THREAD

**Issue by ZhengzhongSun** [JScott/ViveGrip#11]
I have tested your script in many objects and it worked well. But today I try to grab a bird model, this model settings are the same as other objects, but I cannot grab it and it won't become highlight when I touch it. 
![bird](https://cloud.githubusercontent.com/assets/13618800/17017639/bcf5e042-4f66-11e6-9e2e-cdbf7f3f9776.PNG)
Could you give me any advice? Thank you very much!

**Comment by JScott** [comment]
For sure! If it won't highlight then the problem is having the grip's sphere collide with the object. [`NearestObject()`](https://github.com/JScott/ViveGrip/blob/master/Assets/ViveGrip/Scripts/Internal/ViveGrip_TouchDetection.cs#L20) is doing the heavy lifting there, so `OnTriggerEnter` isn't being thrown.

The thing that immediately stands out to me are the two colliders and one of them being a concave mesh collider. I'm not sure what Unity does with two colliders so you might want to chose one and delete the other. You should also make the mesh one convex if you keep it. I expect that one of those 999+ errors is going to be Unity telling you that Rigidbodies colliders in Unity 5 must be concave.

Play around with that a bit and let me know how it goes. If you're still getting trouble then I'll have to try to recreate your scene and see if I can reproduce (and fix) the issue you're seeing.

**Comment by ZhengzhongSun** [comment]
I have removed the mesh collider, but I still cannot move the bird. Could you have a try about the obj?
[bird.zip](https://github.com/JScott/ViveGrip/files/377408/bird.zip)

**Comment by JScott** [comment]
I'll give it a shot when I have some time and let you know how it goes. Are there any components or scripts affecting the bird object other than what I can see in your screenshot?

**Comment by ZhengzhongSun** [comment]
Thank you very much! I don't think there are any components or scripts affecting the bird object .

**Comment by JScott** [comment]
Figured it out: it was your model. I'm not sure why Unity didn't like it but the object centre was way off and it was preventing the colliders from working properly. I recentred it in Blender and brought it back in and it works just fine:

![original centre](http://i.imgur.com/THnj419.png)
![fixed centre](http://i.imgur.com/k6KwUvZ.png)
![bird comparison](https://cloud.githubusercontent.com/assets/25989/17078934/870a7938-50b6-11e6-8f4f-d2fdfee75b7e.gif)

The original you sent is on the left and the fixed OBJ is on the right. For your convenience, here's the model:

[bird_fixed.zip](https://github.com/JScott/ViveGrip/files/379855/bird_fixed.zip)

So many of my problems come from models not exporting properly and Unity doing weird things as a result :)
